### PR TITLE
Create LIT6920CommentJubileesWasan.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
An new work from Abbadie 154.

### CODE REVIEW

File: new/LIT6920CommentJubileesWasan.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6920CommentJubileesWasan" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="am" xml:id="t1">Amharic commentary on Jubilees by Wasan of Goggam</title>
```
Comment (thea-m) [inline new/LIT6920CommentJubileesWasan.xml:8]: en

File: new/LIT6920CommentJubileesWasan.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>According to <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName> , the commentary was dictated for by <persName ref="PRS14332Wasan"/> and written down.
```
Comment (thea-m) [inline new/LIT6920CommentJubileesWasan.xml:33]: I feel that something is missing after "for"

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/LIT6920CommentJubileesWasan.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="am" xml:id="t1">Amharic commentary on Jubilees by Wasan of Goggam</title>
+                <title xml:lang="en" xml:id="t1">Amharic commentary on Jubilees by Wasan of Goggam</title>
```
Comment (thea-m) [inline new/LIT6920CommentJubileesWasan.xml:8]: Sorry, I see it only now, Goǧǧām?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/LIT6920CommentJubileesWasan.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6920CommentJubileesWasan.xml]: sorry, corrected

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]

File: new/LIT6920CommentJubileesWasan.xml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-02-09">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:isAttributedToAuthor" active="LIT6920CommentJubileesWasan" passive="PRS14332Wasan"/>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT6920CommentJubileesWasan.xml:60]: Sorry, do you mean that I have to use `<relation name='dcterms:creator'/>`, or smth else?

[the remark this answers is not in the document]
Comment (eu-genia) [reply new/LIT6920CommentJubileesWasan.xml:60]: even better, yes `dcterms:creator` should be the best option

[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT6920CommentJubileesWasan.xml:60]: ok

File: new/LIT6920CommentJubileesWasan.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6920CommentJubileesWasan.xml]: ok

File: new/LIT6920CommentJubileesWasan.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT6920CommentJubileesWasan.xml]: ok

File: new/LIT6920CommentJubileesWasan.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/LIT6920CommentJubileesWasan.xml:60]: still unchanged

[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply new/LIT6920CommentJubileesWasan.xml]: sorry, now done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**